elastic/integrations
# [O11y][WebSphere Application Server] Update the docker image

- Bug

## What does this PR do?

- This PR updates the docker image for WebSphere Application Server according to latest [releases](https://hub.docker.com/r/ibmcom/websphere-traditional#:~:text=9.0.5.17%2C%209.0.5.17%2Dubi8%2C%209.0.5.17%2Dubi8%2Dppc64le%2C%209.0.5.17%2Dubi8%2Ds390x%2C%209.0.5.17%2Dubi8%2Damd64%2C%20latest)
- This change is required as the older docker image `9.0.5.11` is failing due to some errors mentioned in this [issue](https://github.com/elastic/integrations/issues/8004). Those are fixed in the latest version.

## Checklist


## How to test this PR locally

1. Run `elastic-package stack up -d -v` on the package `websphere_application_server`
2. Run `elastic-package test -v`
3. Check that all the tests are passing



Closes #8004
